src/main/java/no/sikt/sws/WorkspaceStripper.java

```diff
@@ -0,0 +1,31 @@
+package no.sikt.sws;
+
+import nva.commons.core.JacocoGenerated;
+
+import java.util.Arrays;
+
+import static nva.commons.core.attempt.Try.attempt;
+
+@JacocoGenerated
+public class WorkspaceStripper {
+
+    // Remove {workspace}- from responseBody
+    public static String remove(String body, String workspace) {
+        if (body == null) {
+            return null;
+        }
+        return body.replaceAll(workspace + "-","");
+    }
+
+    // replace {index} with {workspace}-{index} from responseBody
+    public static String add(String body, String workspace, String index) {
```

Comment:
1. In the comment you write responseBody, but this is what we actually send as a requestBody to opensearch.
2. Can you delete it, and we can add it later if we need it. Right now it can cause issues for "wrong" index/workspace names